ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-release-ocm-2.11.yaml

```diff
@@ -0,0 +1,152 @@
+base_images:
+  assisted-image-service:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-image-service
+  assisted-installer:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-installer
+  assisted-installer-controller:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-installer-controller
+  assisted-service:
+    name: ocm-2.11
+    namespace: edge-infrastructure
+    tag: assisted-service
+  assisted-service-index:
+    name: assisted-service-index
+    namespace: edge-infrastructure
+    tag: ocm-2.11
```

Comment:
ocm-2.11 needs to be mirrored: https://github.com/openshift/release/blob/master/core-services/image-mirroring/_config.yaml#L444-L455